### ISSUE TITLE
Provide jar file to download

### ISSUE THREAD

**Issue by kiru** [vaadin/framework8-migration-tool#33]
We use [vaadin gradle](https://github.com/johndevs/gradle-vaadin-plugin) plugin.
My experience with this tool so far:
- Check out this repo
- try to build with mvn - ok maven not up-to-date
- import into IntelliJ - run mvn clean first then mvn install
- Ok, all right, go to my project folder and run: 
> java -jar $HOME/.m2/repository/com/vaadin/framework8-migration-tool/8.0-SNAPSHOT/framework8-migration-tool-8.0-SNAPSHOT.jar
- Of course, it did not work:
> Error: Unable to access jarfile $HOME/.m2/repository/com/vaadin/framework8-migration-tool/8.0-SNAPSHOT/framework8-migration-tool-8.0-SNAPSHOT.jar
- Anyway, in the mvn output I find the following output:
>[INFO] Installing C:\framework8-migration-tool\target\framework8-migration-tool-8.0-SNAPSHOT.jar to C:\Users\me\.m2\repository\com\vaadin\framework8-migration-tool\8.0-SNAPSHOT\framework8-migration-tool-8.0-SNAPSHOT.jar
>[INFO] Installing C:\framework8-migration-tool\pom.xml to C:\Users\me\.m2\repository\com\vaadin\framework8-migration-tool\8.0-SNAPSHOT\framework8-migration-tool-8.0-SNAPSHOT.pom
- Copy framework8-migration-tool-8.0-SNAPSHOT.jar to my project folder and just run:
> java -jar framework8-migration-tool-8.0-SNAPSHOT.jar

Some thoughts:
- Provide the final jar file as download ( maybe on vaadin home page or just in this github repo as realse)
- I don't thinks adding it to "vaadin-maven-plugin" is a great idea, because the "mvn vaadin:upgrade8" command is just a once-in-a-project-lifetime call
- A better workflow for me ( and maybe for others as well) would have been: 
-- Go to this repo or vaadin page
-- Download upgrade8 jar file
-- run "java -jar <path-to-above-jar>"
-- Remove upgrad8.jar

**Comment by mvysny** [comment]
I agree with you here. The migration tool actually produces a self-contained jar with all dependencies, so all you need to do is:
```bash
git clone https://github.com/vaadin/framework8-migration-tool
cd framework8-migration-tool
./mvnw -C clean package
```
then:
* copy `target/*.jar` to your project as `upgrade8.jar`,
* run `java -jar upgrade8.jar 8.5.2`
* after that's done, remove `upgrade8.jar`

If you agree, I'll update the README.md of this project in this regard.

**Comment by mvysny** [comment]
Because on every migration project I've been, I needed to add a couple of custom rules, tweak the migration tool a bit here and there. Which is not possible with a distributed jar.

With that being said, we can still publish the jar, but I don't really see the point.

**Comment by kiru** [comment]
I see your point, but I think if this migrations tool is good then it should cover most use cases ( and hence you don't need to spend the time to understand this tool and tweak it ).

**Comment by mvysny** [comment]
You have a point there. We'll need to decide on the release schedule + versioning, then we can do the release.
